docs/guides/zookeeper/README.md

```diff
@@ -0,0 +1,40 @@
+---
+title: ZooKeeper
+menu:
+  docs_{{ .version }}:
+    identifier: zk-readme-zookeeper
+    name: ZooKeeper
+    parent: zk-zookeeper-guides
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+url: /docs/{{ .version }}/guides/zookeeper/
+aliases:
+  - /docs/{{ .version }}/guides/zookeeper/README/
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+## Supported ZooKeeper Features
+| Features                                                   | Community | Enterprise |
```

Comment:
No need for separate | Community | Enterprise | columns. we can keep a Availability column